staging/src/k8s.io/client-go/pkg/api/types.go

```diff
@@ -2213,6 +2213,17 @@ type PodSpec struct {
 	// file if specified. This is only valid for non-hostNetwork pods.
 	// +optional
 	HostAliases []HostAlias
+	// If specified, indicates the pod's priority. "SYSTEM" is a special keyword
+	// which indicates the highest priority. Any other name must be defined by
+	// creating a PriorityClass object with that name.
+	// If not specified, the pod priority will will be default or zero if there is
+	// no default.
+	// +optional
+	PriorityClassName string
+	// The priority value which is resolved by Priority Admission Controller from
+	// PriorityClassName. The higher the value, the higher the priority.
+	// It must not be specified by user.
```

Comment:
Documenting this in terms of an admission plugin reads really strangely. The API field should not require an admission plugin to make sense. If the admission plugin and the behavior described is required, then it probably shoudn't be an admission plugin, and should be part of the rest handler.